webapp/templates/accounts/member_list.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html' %}
+{% load bootstrap3 %}
+{% block title %}Ledenlijst{% endblock %}
+{% block content %}
+    <h1 class="page-header">Ledenlijst</h1>
+
+    <!-- Table -->
+    <table class="table table-hover" id="accounts-table">
+        <thead>
+        <tr>
+            <th>Naam</th>
+            <th>Rijbewijs</th>
+            <th>E-mail</th>
+            <th>Telefoon</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for account in object_list %}
+        <tr class="account">
+            <td class="col-md-2">{{ account.get_full_name }}</td>
+            <td class="col-md-1">{{ account.userprofile.has_drivers_license|yesno:"Ja,Nee" }}</a></td>
+            <td class="col-md-2">{{ account.email }}</td>
+            <td class="col-md-2">{{ account.userprofile.phone_number }}</td>
+        </tr>
+        {% endfor %}
+        </tbody>
+    </table>
+{% endblock %}
+
+{% block javascript %}
+    <script>
+        $(document).ready(function() {
```

Comment:
Can you add localization for the search field (like https://github.com/rikva/voko/blob/master/webapp/templates/ordering/product_list.html#L135)